java/src/jmri/jmrix/mqtt/MqttTurnout.java

```diff
@@ -38,10 +39,11 @@ public void setParser(MqttContentParser<Turnout> parser) {
     }
         
     MqttContentParser<Turnout> parser = new MqttContentParser<Turnout>() {
-        private final static String closedText = "CLOSED";
-        private final static String thrownText = "THROWN";
-        private final static String unknownText = "UNKNOWN";
-        private final static String inconsistentText = "INCONSISTENT";
+        // public for scripting
+        public final static String closedText = "CLOSED";
+        public final static String thrownText = "THROWN";
+        public final static String unknownText = "UNKNOWN";
+        public final static String inconsistentText = "INCONSISTENT";
```

Comment:
Can these really be reached from scripting? They're static on an anonymous class.

Reply:
I think the constants can stay in this file, but moved to the outer class MqttTurnout.